module/solutions/helper.js

```diff
@@ -2659,37 +2689,95 @@ module.exports = class SolutionsHelper {
 		userId,
 		pageSize,
 		pageNo,
-		search,
+		search = '',
 		filter,
 		surveyReportPage = '',
-		currentScopeOnly = false,
-		entityId = ''
+		currentScopeOnly = false
 	) {
 		return new Promise(async (resolve, reject) => {
 			try {
 				currentScopeOnly = UTILS.convertStringToBoolean(currentScopeOnly)
-				let assignedSolutions = await this.assignedUserSolutions(
-					solutionType,
-					userId,
-					search,
-					filter,
-					'',
-					'',
-					entityId
-				)
-				if (!assignedSolutions.success) {
+				if (currentScopeOnly && process.env.SUBMISSION_LEVEL == 'ENTITY') {
```

Comment:
@Prajwal17Tunerlabs tekdi won't encounter this use case?

Reply:
currentScopeOnly was introduced to only fetch those projects whose scope matches given in the req.body. But when entityId comes into picture we should be able to fetch those projects even if their scope doesnt match. This is contradictory.